### PR TITLE
[fb] Changed file system name in permission_actions

### PR DESCRIPTION
**Pull request by tarunjangid**
## What changes were proposed in this pull request?

- We changed the file system names to include more detailed information for end-users.

## How was this patch tested?

- manual testing

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: apps/filebrowser/src/filebrowser/settings.py [hunk 1]
```diff
@@ -28,9 +28,9 @@
...
-  (PERMISSION_ACTION_S3, "Access to S3 from filebrowser and filepicker."),
-  (PERMISSION_ACTION_ADLS, "Access to ADLS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_ABFS, "Access to ABFS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_GS, "Access to GS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_OFS, "Access to OFS from filebrowser and filepicker.")
+  (PERMISSION_ACTION_S3, "Access to AWS S3 from filebrowser and filepicker."),
+  (PERMISSION_ACTION_ADLS, "Access to Azure Data Lake Storage (ADLS) from filebrowser and filepicker."),
+  (PERMISSION_ACTION_ABFS, "Access to Azure Blob File System (ABFS) from filebrowser and filepicker."),
+  (PERMISSION_ACTION_GS, "Access to Google File System (GS) from filebrowser and filepicker."),
+  (PERMISSION_ACTION_OFS, "Access to Ozone File System (OFS) from filebrowser and filepicker.")
```
Comment (Harshg999) [inline apps/filebrowser/src/filebrowser/settings.py:35]: Nice!

**Review by Harshg999 — APPROVED** [review]
LGTM!

File: apps/filebrowser/src/filebrowser/settings.py [hunk 2]
```diff
@@ -28,9 +28,9 @@
 
 
 PERMISSION_ACTIONS = (
-  (PERMISSION_ACTION_S3, "Access to S3 from filebrowser and filepicker."),
-  (PERMISSION_ACTION_ADLS, "Access to ADLS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_ABFS, "Access to ABFS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_GS, "Access to GS from filebrowser and filepicker."),
-  (PERMISSION_ACTION_OFS, "Access to OFS from filebrowser and filepicker.")
+  (PERMISSION_ACTION_S3, "Access to AWS S3 from filebrowser and filepicker."),
```
Comment (agl29) [inline apps/filebrowser/src/filebrowser/settings.py:31]: @tarunjangid 
for similarity->  Amazon Web Services (AWS)

File: apps/filebrowser/src/filebrowser/settings.py (hunk 1 above)
Comment (tarunjangid) [reply apps/filebrowser/src/filebrowser/settings.py]: Thanks, Harsh!

**Pull request merged**